src/modules/navigator/rtl_params.c

```diff
@@ -120,7 +121,8 @@ PARAM_DEFINE_INT32(RTL_TYPE, 0);
 /**
  * Half-angle of the RTL cone.
  *
- * Defines the half-angle of the cone which defines the vehicle RTL behavior.
+ * Defines the half-angle of a cone centered around the home position which
```

Comment:
which -> that

Do we ever link to docs? Could choose to link to new PR info on this.